### PR TITLE
Resolve #3558

### LINKED ISSUE #3558: How to run single package tests.
Whole suite takes ages, which is frustrating.

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ several.  In general, the test executable for
 `{Cabal,cabal-install}:$TESTNAME` will be stored at
 `dist-newstyle/build/{Cabal,cabal-install}-$VERSION/build/$TESTNAME/$TESTNAME`.
 
+To run a single test, use `-p` which applies a regex filter to the test names.
```
Comment (23Skidoo) [inline README.md]: I'd mention `--test-option` and add an example.

Comment (phadej) [inline README.md]: `README.md` talks about `cabal new-build` only, until there is `new-test`, i'd postpone adding `--test-option`.

Comment (phadej) [inline README.md]: `README.md` talks about `cabal new-build` only, so I'd postpone adding `--test-option`, until we talk have `new-test`.

Comment (23Skidoo) [inline README.md]: OK.

**Comment by 23Skidoo** [comment]
Merged, thanks!

**Pull request merged**